apps/ca.c

```diff
@@ -1558,7 +1557,7 @@ static int do_body(X509 **xret, EVP_PKEY *pkey, X509 *x509,
 
             if (push != NULL) {
                 if (!X509_NAME_add_entry(subject, push, -1, 0)) {
-                    X509_NAME_ENTRY_free(push);
+                    /* useless X509_NAME_ENTRY_free(push); */
```

Comment:
Why comment this out? The original code just looks wrong AFAICT - shouldn't we just delete it?